rpa-manager/modules/ROOT/pages/settings-connect.adoc

```diff
@@ -0,0 +1,83 @@
+= Connecting to RPA Processes from Salesforce Products
+:page-notice-banner-message: You can now use Connected Apps for a more secure and scalable authentication method than User API Keys.
+
+RPA Manager requires authentication to integrate RPA processes into other Salesforce products. For example, you can use your API key to configure an RPA connector in a MuleSoft Composer flow.
+
+For security reasons, you can only see and copy an API key on creation.
+
+== Before You Begin
+
+* Ask an Anypoint Access Management administrator to assign you the required permissions:
++
+[cols="1,1,1"]
+|===
+|*Action* |*RPA Permission* | *Deprecated RPA Permission*
+
+|View configured settings for your organization
+|RPA Administrator
+|Organization Management Open
+
+|Create, edit, and delete API keys
+|RPA Administrator
+|User API Key Create, User API Key Edit, User API Key Delete
+
+|===
+
+* Users must have an Anypoint account to associate the API key with.
+* You must have a secure way to transfer the API Key to the user.
+* Users must have a secure place accessible to store their API key.
+
+[[create-an-api-key-for-a-user]]
+== Create a User API Key
+
+Create an API key for a user to integrate an RPA process into another Salesforce product. After expiration, the integrated process instance can no longer connect to RPA Manager from other Salesforce products. Edit the API key to move the expiration date. For security reasons, RPA Manager shows the API Key only during creation. Copy the API Key and securely send it to the user. If the user lose an API Key, create a new one.
+
+To create a user API key: 
+
+. Open the *User API Keys* view of the *User Management* module.
+. Click *Create*.
+. [[form-createuserapikey]] Complete the form *Create User API Key*:
+* *User*:
++
+Select a user whose identity and permissions are used to connect from another Salesforce products to RPA Manager.
+* *Expiration Date*:
++
+Select an expiration date for the user API key from the calendar.
+. Click *OK*.
+. In the window *Key Created!*, click *Copy to clipboard*.
```

Comment:
```suggestion
. In the window *Key Created*, click *Copy to clipboard*.
```

Reply:
Our standard is to exclude punctuation in names or label references.
See https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference, under UI elements.

Reply:
Created bug [W-13871084](https://gus.lightning.force.com/a07EE00001XRYfSYAX) for dev

Reply:
and removed ! in docs